pkg/target/http.go

```diff
@@ -358,10 +310,11 @@ func (f HTTPTargetAdapter) ProvideDefault() (interface{}, error) {
 			Invalid:    []Rule{},
 			SetupError: []Rule{},
 		},
-		IncludeTimingHeaders: false,
+		IncludeTimingHeaders:       false,
+		RejectionThresholdInMillis: 200,
```

Comment:
150 actually turned out to be the best configuration in the end. Can change the default please? (sorry)